docs/content/engines/hive.md

```diff
@@ -213,6 +218,35 @@ OK
 
 ```
 
+* To create paimon tables that do not exist, you can create them as external and internal tables in Hive. Run the following Hive SQL in Hive CLI.
+
+```sql
+-- Assume that paimon-hive-connector-{{< version >}}.jar is already in auxlib directory.
+-- Let's create a new table that doesn't exist in paimon.
+-- Need to specify the location to the path of table.
+
+CREATE EXTERNAL TABLE hive_test_table(
+    a INT COMMENT 'The a field',
+    b STRING COMMENT 'The b field'
+)
+STORED BY 'org.apache.paimon.hive.PaimonStorageHandler'
+LOCATION '/path/to/table/store/warehouse/default.db/hive_test_table';
+
+-- Insert records into hive_test_table
+
+INSERT INTO hive_test_table VALUES (1, 'Paimon');
```

Comment:
Can this table be visible in Flink Paimon Catalog with hive metastore?

Reply:
This table is visible, 
document link: 
https://github.com/apache/incubator-paimon/blob/7ec22998b2d4aa3d97fbc5cfc1683ef3daf29ad3/docs/content/engines/hive.md?plain=1#L135-L163

test case:
https://github.com/apache/incubator-paimon/blob/7ec22998b2d4aa3d97fbc5cfc1683ef3daf29ad3/paimon-hive/paimon-hive-connector-common/src/test/java/org/apache/paimon/hive/HiveCatalogITCaseBase.java#L268-L280